new_latte/coffee_latte.py

```diff
@@ -0,0 +1,24 @@
+from new_coffee.coffee import *
```

Comment:
Wildcard imports (from <module> import *) should be avoided
https://www.python.org/dev/peps/pep-0008/#imports